### DISCUSSION TITLE
輸入文字就能生成高解析圖片！OpenAI 發表新版 DALL·E 2 AI 系統

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#552] [科技新知]
AI 人工智慧的發展一日千里，並在許多方面展現超越人類的能耐，AI 不但打敗了世界棋王，也擊潰電競冠軍團隊。不僅如此，AI 還會寫文章，由 OpenAI 研究實驗室推出的 GPT-2 及 GPT-3 文字產生預訓練語言模型，因為能書寫出媲美人類文筆的文章，因而淪為撰寫假新聞的利器。如今，OpenAI 推出新一代的 DALL·E 2 系統，只要透過一段描述文字便能叫 AI 幫你產生各種圖片。
去年 1 月 Open AI 推出的DALL·E，可將使用者輸入文字轉變成生動的超現實主義圖片。例如，使用者可以透過文字描述，要求 DALL·E 產生一張太空人在外太空騎馬的圖片，或是兩隻泰迪熊在月球上致力新 AI 研究的圖片，所以其超現實的程度甚至媲美超現實主義畫家達利（Salvador Dalí）。「DALL·E」這個字就是達利與迪士尼電影《瓦力》（WALL-E）機器人這兩個名字的組合。

https://technews.tw/2022/04/10/openai-dall%c2%b7e-2/

**Comment by ThomasMeng0515** [comment]
這樣的模型確實會有產出假新聞的疑慮，但好險文中有提及，最新的系統具有保護機制，可以避免一些個人隱私，或是令人敏感的主題，將能製造假新聞的可能性降低。

**Comment by HuanYuuuuuu** [comment]
感覺如同deep fake 一樣 如果被有心人士拿來使用就會造成許多假消息的散布

**Comment by Sazmafuji** [comment]
我們需要越來越需要注意假新聞，但這些技術搭配deep fake等技術，想必假新聞也會被以為是真得

**Comment by bloodtrail0511** [comment]
可以把自己做的奇葩的夢變成圖片，太酷了吧

**Comment by PoooYen** [comment]
終於知道瓦力的由來了

**Comment by cordydeng** [comment]
這個技術感覺好有趣，或許可以成為藝術家作畫時的參考資料

**Comment by sys511613** [comment]
用了這個技術就可以很容易的把想法變成圖片，對生產力也會有很大的提升

**Comment by Rosancl** [comment]
雖然有防護機制，但還是仍得小心被有心人士利用

**Comment by awkward-willy** [comment]
或許有了這個技術能讓更多天馬行空的想法被圖像表達出來
不過相關防護措施的安全與否想必在未來會形成一大考驗

**Comment by yuhao112** [comment]
太厲害了!感謝分享

**Comment by d26488399** [comment]
太厲害了，感覺未來只要寫完一個故事就能自動轉成動畫

**Comment by ghost** [comment]
我覺得還有一段漫長的路要走 但希望未來真能實現\

**Comment by henrywen547** [comment]
雖然會有散布假消息的疑慮，但是這個技術好厲害啊，好期待能生成顛覆思維的圖片!

**Comment by HaoFish123** [comment]
感謝分享，可以找到許多圖片資源。

**Comment by liangyinglly** [comment]
希望人們能把它運用在增進人類福祉的地方

**Comment by etin2468** [comment]
AI在藝術創作方面的能力也變得很強了呢

**Comment by Lynn12378** [comment]
AI進步讓人不知是喜是優，連藝術方面都有可能被取代的話，將來競爭力感覺會越來越強
